app/templates/components/crate-readme.hbs

```diff
@@ -1 +1 @@
-{{{rendered}}}
+{{html-safe rendered}}
```

Comment:
Nit: Please keep the newline at the end of file here. Many editors will add it back, resulting in spurious diffs. (Probably not so relevant for this particular file, but well.)